addon/locale/zh_CN/LC_MESSAGES/nvda.po

```diff
@@ -83,7 +83,7 @@ msgstr "面部位置合适"
 
 #: addon\globalPlugins\webcamHelper\__init__.py:110
 msgid "Starts continuous webcam analysis, press escape to cancel"
-msgstr "已开始摄像头检测，按 ESC 取消。"
+msgstr "已开始摄像头检测，按 ESC 取消"
```

Comment:
is this change to chinese translation intentional?

Reply:
Yes. I made it to avoid error in poedit. It's found a discrepancy between the source text and the translated text which I fixed.